### PR TITLE
DDP-4347: Update Osteo carousel

### PR DESCRIPTION
**Pull request by fat23cat**
- Added 4 of 5 pictures. On Friday's Process meeting we decided that we'll go with 4 pics, then add 5th(when it will be available)
- Images optimized, placeholders removed

### CODE REVIEW

File: ddp-workspace/projects/ddp-osteo/src/assets/i18n/en.json [hunk 1]
```diff
@@ -60,30 +60,30 @@
             {
                 "id": 0,
                 "title": "Touring the Genomics Platform",
-                "text": "Members of the Count Me In team and the Osteo Project’s Patient Advisory Council tour the Genomics Platform at the Broad Institute.",
-                "image": "assets/images/osteo-team.png",
-                "alt": "Osteo team"
+                "text": "Members of the Count Me In Team and the OSP Project Advisory Council tour the Genomics Platform at the Broad Institute.",
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-osteo/src/assets/i18n/en.json:63]: "OSP Project" does not sound right to me. I thought OSP stood for "Osteosarcoma Project", so OSP Project would not make sense. You got this text from Erin? Maybe she can confirm?

File: ddp-workspace/projects/ddp-osteo/src/assets/i18n/en.json [hunk 2]
```diff
@@ -60,30 +60,30 @@
...
             },
             {
                 "id": 2,
-                "title": "Nulla placerat justo vel vestibulum rutrum",
-                "text": "Integer vitae mattis enim. Nam risus sem, blandit et orci nec, volutpat tincidunt ipsum. Donec pellentesque dapibus tincidunt.",
-                "image": "assets/images/placeholder2.jpg",
-                "alt": "Osteo team"
+                "title": "Project Advisory Council",
+                "text": "We are so grateful to the PACs that worked with us to build the OSP. Every page of this site was built with their partnership and with this community in mind. Learn more about each one of them on the About Us page.",
+                "image": "assets/images/pac.jpg",
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-osteo/src/assets/i18n/en.json:78]: We should confirm this one too. As far as I know, there is only on Project Advisory Council with multiple members, but we refer to PACs. Again, could be right, but we should double-check.

**Review by MocanaAtBroad — APPROVED** [review]
Changes look good and work well, so approving.  PR correctly copied language from provided spreadsheet, but need to double-check anyways.

File: ddp-workspace/projects/ddp-osteo/src/assets/i18n/en.json (hunk 1 above)
Comment (eringwozdz) [reply ddp-workspace/projects/ddp-osteo/src/assets/i18n/en.json:63]: [OSP] is one thing, [Project Advisory Council] aka PAC is another thing.

File: ddp-workspace/projects/ddp-osteo/src/assets/i18n/en.json (hunk 2 above)
Comment (eringwozdz) [reply ddp-workspace/projects/ddp-osteo/src/assets/i18n/en.json:78]: For Osteo we have two PACs. One with osteo patients, and one with parents who have lost their children to osteo.

**Pull request merged**